src/features/content-answers/ContentAnswersModal.tsx

```diff
@@ -62,6 +63,7 @@ const ContentAnswersModal = ({ api, currentUser, file, isOpen, onRequestClose }:
 
     const handleOnAsk = useCallback(
         async (prompt: string) => {
+            onAsk();
```

Comment:
this also highlights if the first question returns an error, is that okay? or do we want to move until after a successful response?

Reply:
the other one does that also, so it meets parity